package.json

```diff
@@ -26,16 +26,19 @@
     "eslint-config-webrtc": "^1.0.0",
     "express": "^4.14.1",
     "faucet": "0.0.1",
+    "geckodriver": "1.4.0",
```

Comment:
pinning should not be required... also this should pull from utils?

Reply:
Its strange, when pulled from utilities, it's installed it says, but does not execute all the steps and thus is not in node_modules nor in .bin but it's in the tree if I do npm ls.....

Reply:
Aha, I know why, utilities do not have geckodriver listed as a devDeps, just a normal deps.

Reply:
if its a dep it will be puled and installed. if its a dev-dep it wont. Lets get rid of the utils and pull everything directly?